src/mlpack/methods/random_forest/random_forest_main.cpp

```diff
@@ -89,7 +89,7 @@ BINDING_EXAMPLE(
     "predictions for each point to " + PRINT_DATASET("predictions") + ", one "
     "could call "
     "\n\n" +
-    PRINT_CALL("random_forest", "input_model", "rf_model", "test", "test_set",
+    PRINT_CALL("random forest", "input_model", "rf_model", "test", "test_set",
```

Comment:
This needs to remain as `random_forest`---using `PRINT_CALL()` should use the name of the binding itself.  Can you change this back please?